src/Summoner/Project.hs

```diff
@@ -45,7 +45,7 @@ generateProject projectName Config{..} = do
     -- License creation
     fetchedLicense <- fetchLicense license
     year <- currentYear
-    let licenseText = customizeLicense license fetchedLicense nm year
+    let licenseText = customizeLicense license (unLicense fetchedLicense) nm year
```

Comment:
Hmm, I guess it's good to make `customizeLicense` receive `License` as well and unwrap it only there. We have three `Text` types here otherwise, which is not very good. Could you fix this as well? :slightly_smiling_face: